index.html

```diff
@@ -1978,12 +1978,14 @@ <h3>Terms of Use</h3>
         <h3>Evidence</h3>
 
         <p>
-Evidence is used by an <a>issuer</a> to determine whether or not to issue
-a <a>credential</a>. For example, an issuer might check physical documentation
-provided by the <a>subject</a> or might perform a set of background checks
-before issuing the credential. In certain scenarios, this information is useful
-to the <a>verifier</a> when determining the risk associated with relying on a
-given credential.
+Evidence is used by an <a>issuer</a> to inform the <a>verifier</a> of the basis
+for the <a>verifiable credential</a>. This could be used by the <a>verifier</a>
+to establish the confidence with which it relies on the claims in the
+<a>verifiable credential</a>. For example, an issuer might check physical 
+documentation provided by the <a>subject</a> or might perform a set of background
+checks before issuing the credential. In certain scenarios, this information is
+useful to the <a>verifier</a> when determining the risk associated with relying
+on a given credential.
```

Comment:
The language is an improvement. I'd like us to go further and mention Level of Assurance processes and links to physical documentation in the evidence field. The description we have at present is vague and we should provide more examples of the types of things you can put in the evidence field. I'm happy to provide concrete extra examples (downloadable links, LoA)... just thinking out loud, but hopefully this PR can add the language and I'll add the examples if we can get this PR in.

Reply:
+1 see updates.